cli/dcoscli/node/main.py

```diff
@@ -24,9 +25,16 @@ def main():
 @decorate_docopt_usage
 def _main():
     util.configure_process_from_environ()
+    argv = sys.argv[1:]
```

Comment:
I'd much rather let `docopt.docopt()` decide what `argv` is, but I'm not sure how else to add this deprecation notice without adding `--master` back to the usage string.